From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 3 Sep 2010 00:18:16 -0400
Subject: [net] vxge: remove trailing space in messages
Message-id: <20100903001815.16045.17296.stgit@localhost6.localdomain6>
Patchwork-id: 28046
O-Subject: [RHEL5.6 BZ608598 PATCH 05/15] vxge: remove trailing space in messages
Bugzilla: 608598
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Bob Picco <bpicco@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>

from upstream commit 2381a55c88453d3f29fe62d235579a05fc20b7b3

diff --git a/drivers/net/vxge/vxge-main.c b/drivers/net/vxge/vxge-main.c
index cdc6b23..4c0131f 100644
--- a/drivers/net/vxge/vxge-main.c
+++ b/drivers/net/vxge/vxge-main.c
@@ -4172,7 +4172,7 @@ vxge_probe(struct pci_dev *pdev, const struct pci_device_id *pre)
 			ret = pci_enable_sriov(pdev, max_config_dev - 1);
 			if (ret)
 				vxge_debug_ll_config(VXGE_ERR,
-					"Failed to enable SRIOV: %d \n", ret);
+					"Failed to enable SRIOV: %d\n", ret);
 	}
 
 	/*
